.githooks/pre-commit

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+
+# Check copyright notice
+files=`git diff --cached --name-only`
+year=`date +%Y`
+
+for f in $files; do
+    head -10 $f | grep -i "Copyright (c)" 2>&1 1>/dev/null || continue  # Copyright not present in this file
+
+    if ! grep -i -e "$year, NVIDIA CORPORATION" $f 2>&1 1>/dev/null; then
+        need_update_year="$need_update_year $f"
```

Comment:
I guess files with the spaces are not expected here (I don't think we care about them ever).